frontend-friends/src/components/textfield/textfield.scss

```diff
@@ -15,6 +15,40 @@
     }
 }
 
-input:required {
-    color: red;
+.input-text-field {
+    position: relative;
+
+    input {
+        all: unset;
+        border-radius: 6px;
+        height: 50px;
+        background-color: colors.$text-field-background;
+        border: 1px solid colors.$text-field-stroke;
+        box-sizing: border-box;
+        padding: 0 20px;
+        //padding: 10px;
+        font-size: 25px;
+    }
+    input:valid + .placeholder {
+        display: none;
+    }
+  }
+ 
+  
+.placeholder {
```

Comment:
Funderar på om borde flytta in .placeholder till innanför .input-text-field för att inte få oönskat beteende om klassen placeholder skulle användas på annan plats i applikationen.

Reply:
Har fixat det så att de ligger innanför blir dock lite duplicerad kod.